docker-compose-build.yaml

```diff
@@ -11,7 +11,7 @@ services:
         - PY_VER
         - DISTRO
         - IMAGE
-    image: datajoint/datajoint:${DJ_VERSION}
+    image: datajoint/datajoint:0.13.7
```

Comment:
Please revert this change. This previously extracted the current version to tag the docker image appropriately for the official `datajoint` image.

Reply:
Apologies, was testing and forgot to revert. will do in the next commit.